Snowflake.Data/Core/FileTransfer/StreamPair.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+
+namespace Snowflake.Data.Core.FileTransfer
+{
+    internal class StreamPair: IDisposable
```

Comment:
Changes in HttpRequestMessage seem not required and this class complicates the code a bit. I would revert this

Reply:
my bad, not httprequestmessage but SFRemoteStorageUtil.cs

Reply:
We agreed it is necessary